src/Objs/Redirect/RedirectComponent.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react'
+import {
+  ensureUserIsLoggedIn,
+  isUserLoggedIn,
+  provideComponent,
+  urlFor,
+} from 'scrivito'
+import { InPlaceEditingPlaceholder } from '../../Components/InPlaceEditingPlaceholder'
+import { Redirect } from './RedirectObjClass'
+
+provideComponent(Redirect, ({ page }) => {
+  const requireLogin = page.get('requireLogin')
+  const link = page.get('link')
+  const url = link && urlFor(link)
```

Comment:
I think it would be better to move this to `useEffect` and use load instead. Otherwise the effect might be trigger multiple times, if a newer version of the URL (e.g. with a full slug, after loading the actual obj) is available.